HomeWorkIntro/ThirdTask/main.css

```diff
@@ -13,47 +13,46 @@ body{
   background-color: #F5F5F5;
 }
 
-#top{
+.top{
   background-color: #800080;
-  left:0px;
-  top: 0px;
-  right:0px;
-  height: 130px;
+  width: 100%;
+  height: 15vh;
+  display: block;
 }
 
-#center{
+.center{
+
   background-color: #FF0000;
-  padding: 60px 80px;
-  left: 20%;
-  top: 130px;
-  right: 5%;
-  bottom: 0px;
-  overflow: hidden;
+  min-height: 85vh;
+  width: 60%;
+  overflow: auto;
+  margin: 0 auto;;
 }
 
 .window{
   background-color: #0000FF;
-  position: relative;
   margin-bottom: 45px;
-  height: 220px;
-}
-
-.yellow, .green{
-  position: absolute;
+  height: 35vh;
+  width: 70%;
+  display: block;
+  margin: 40px auto;
 }
 
 .yellow{
   background-color: #FFFF00;
-  left: 24px;
-  top: 22px;
-  width: 26%;
-  height: 115px;
+  float: left;
+  width: 30%;
+  height: 20vh;
+  margin-top: 20px;
+  margin-left: 15px;
```

Comment:
It is better to set margin left in percentage, in this way it will look good on any screen. Now it looks not so good on a big screen (the distance between yellow and green element is too big).